src/Dsl.php

```diff
@@ -20,7 +20,7 @@ function describe($description, callable $fn)
  */
 function context($description, callable $fn)
 {
-    describe($description, $fn);
+    Context::getInstance()->addSuite($description, $fn);
```

Comment:
Are we just favoring explicitness here? I always kind of viewed context and all of its forms as just an alias for describe. Why not just have it delegate to the describe function?